Gems/Atom/RPI/Code/Source/RPI.Edit/Material/MaterialUtils.cpp

```diff
@@ -206,9 +206,13 @@ namespace AZ
             
             bool BuildersShouldFinalizeMaterialAssets()
             {
-                // We default to the faster workflow for developers. Enable this registry setting when releasing the
-                // game for faster load times and obfuscation of material assets.
-                bool shouldFinalize = false;
+                // Disable this registry setting to improve iteration times when making changes to widely used shaders and material types,
+                // like standard PBR, that require a large number of model assets to be reprocessed by the AP. Disabling finalization will
+                // also disable build dependencies between materials and material types. Without those dependencies in place, loading and
+                // reloading material assets will require special handling because typical asset notifications will not be sent when
+                // dependencies are changed. Before finalization is disabled by default, we should explore options for handling dependencies
+                // on standard PBR differently at the model builder level.
```

Comment:
I think we should indicate that we'd like to remove this option entirely if possible.
```suggestion
                // dependencies are changed. Before, this option was disabled by default because of the long iteration times, but caused hot reload problems
                // so we enabled it again. We should explore options for handling dependencies on standard PBR differently at the model builder level,
                // and hopefully improve the iteration times that way. In that case, we should come back and remove the deferred-finalize option entirely.
```